src/components/internal/DataViewViewSaveDrawerContent.jsx

```diff
@@ -32,10 +32,29 @@ function DataViewViewSaveDrawerContent(props) {
 		...props.data,
 		type: (props.allowSharedViewSave === true) ? props.data.type : "mine"
 	});
+	const [isViewDuplicated, setIsViewDuplicated] = useState(false);
+	const [snackbarLabel, setSnackbarLabel] = useState('');

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9648812)
```suggestion
	const [snackbarLabel, setSnackbarLabel] = useState("");
```